steam/http.py

```diff
@@ -209,8 +209,8 @@ async def login(self, username: str, password: str, shared_secret: str | None) -
                     "Cannot perform redirects after login. Steam is likely down, please try again later."
                 )
 
-            for url in resp["transfer_urls"]:
-                await self.post(url=url, data=data)
+            self._session.cookie_jar.update_cookies(self._session.cookie_jar.filter_cookies(URL.COMMUNITY), URL.STORE)
+            self._session.cookie_jar.update_cookies(self._session.cookie_jar.filter_cookies(URL.COMMUNITY), URL.HELP)
```

Comment:
There's no point calling filter_cookies multiple times here
```suggestion
			community_cookies = self._session.cookie_jar.filter_cookies(URL.COMMUNITY)
            self._session.cookie_jar.update_cookies(community_cookies, URL.STORE)
            self._session.cookie_jar.update_cookiescommunity_cookies, URL.HELP)
```

Reply:
read the first comment I wrote above

Reply:
Creating a copy of every cookie rather than refiltering is probably the best way to do this after performing some timeits.

Reply:
how would i go about doing that?

Reply:
This is what i get when i re-use the variable which is kind of strange. they're modifying the object?

first is help
second is store

![August-15-2021 19-21-38](https://user-images.githubusercontent.com/42787085/129495827-14bf156d-09c3-488f-8475-0bb734321146.png)

Reply:
> Creating a copy of every cookie rather than refiltering is probably the best way to do this after performing some timeits.

like using .copy() when passing it in the for loop?

Reply:
I'm not entirely sure as to why update_cookies mutates the cookies passed, yeah you should just .copy() community_cookies

Reply:
bro i've tried copying this like 30 times 😭 
this part is confusing me idk why it works for one and not the other. 

If you figure it out feel free to just commit it yourself

![image](https://user-images.githubusercontent.com/42787085/129496293-2d09bb79-9e03-4ad1-98ff-1e25312c9901.png)

Reply:
You must need to do a deep copy, but it doesn't matter that much just re-filter the cookies

Reply:
lowkey i totalalllyyyy forgot that lib existed because copy() has never done me wrong haha

Reply:
hope that's faster! but either way it will 100% be faster then a web request